### DISCUSSION TITLE
When you remap Cmd on an external keyboard in MacOS, Ghostty doesn't respect it

### DISCUSSION THREAD

**Discussion by TimelyToga** [ghostty-org/ghostty#3794] [Q&A] [answered]
I am using a "Windows" external keyboard so the position of `Windows` and `Alt` buttons are swapped from how Mac has `Cmd` and `Option`. But Ghostty doesn't respect this remapping, so I need to press `Windows` + `Backspace` instead of `Remapped-Alt-as-Cmd` + `Backspace` to delete a whole line. 

![image](https://github.com/user-attachments/assets/6884efa1-cc9d-471b-91dc-0c26df744406)

**Comment by mitchellh** [comment, marked as answer]
I use a remapped keyboard and this doesn't happen to me. Its been long enough without further reports or upvotes that I'm going to close this due to inactionability. if this still happens, please open a new discussion with additional details to reproduce.
